cmake/kokkoskernels_features.cmake

```
@@ -13,12 +13,6 @@ KOKKOSKERNELS_ADD_OPTION(
   OFF
```

Can we change the default to enable supernodal_sptrsv now?

```suggestion
  ON
```

---

Maybe, @srajama1 or @ndellingwood has some inputs..

---

@e10harvey. I changed it to enable this by default. Now, the unit-test tests it by default, and this is the results of spot-check on Weaver:
> #######################################################
PASSED TESTS
#######################################################
cuda-9.2.88-Cuda_Serial-release build_time=1106 run_time=386
gcc-7.2.0-OpenMP-release build_time=270 run_time=74
gcc-7.2.0-OpenMP_Serial-release build_time=355 run_time=209
gcc-7.2.0-Serial-release build_time=248 run_time=63

---

Thank you, @iyamazaki ! I am ready to merge once the link error in the `light_LayoutRight` CI job is resolved.